Gemfile

```diff
@@ -1,2 +1,3 @@
 source 'https://rubygems.org'
-gemspec
\ No newline at end of file
+gemspec
+gem "pry", :group => :development
```

Comment:
yes, added this for testing. I like that we have it somehow here, but is just my taste for debugging :-P